pyproject.toml

```diff
@@ -9,15 +9,11 @@ show_missing = true
 minversion = "6.0"
 log_cli_level = "INFO"
 
-# Formatting tools configuration
-[tool.black]
-line-length = 99
-target-version = ["py38"]
-
 # Linting tools configuration
 [tool.ruff]
 line-length = 99
 extend-exclude = ["__pycache__", "*.egg_info"]
+target-version = "py38"
```

Comment:
did we switch the 1.32 branch to ruff?

Reply:
[yes we did... nice](https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/blob/release_1.32/tox.ini#L27-L33)